extensions/database-documentation/README.md

```diff
@@ -0,0 +1,13 @@
+# README
+## This is the README for your extension "database-documentation"
```

Comment:
Either update this with correct content for this extension or make an issue for updating it so we can track getting it updated

Reply:
Issue for tracking: https://github.com/microsoft/azuredatastudio/issues/23415